### PR TITLE
add triggers video

### PR DESCRIPTION
**Pull request by taroface**
Add the tutorial video by Rob Reid to the triggers doc.

### CODE REVIEW

**Comment by taroface** [comment]
@florence-crl, this can wait until Monday!

File: src/current/v24.3/triggers.md [hunk 1]
```diff
@@ -508,6 +508,12 @@ In the following example, a combination of `BEFORE` and `AFTER` triggers is used
 	1. The row value fails the conditional check in `ensure_minimum_wage`, and raises an exception.
 	1. The `ERROR` message is printed and the SQL operation is aborted before the `give_bonus` function is executed.
 
+### Video demo
+
+For a deep-dive demo on triggers, play the following video:
```
Comment (florence-crl) [inline src/current/v24.3/triggers.md:513]: nit, for consistency with references to other video links:
```suggestion
For a deep-dive demo on triggers, watch the following video:
```

**Review by florence-crl — APPROVED** [review]
lgtm with non-blocking nit

File: src/current/v24.3/triggers.md (hunk 1 above)
Comment (taroface) [reply src/current/v24.3/triggers.md:513]: I changed the wording here for the same reason others have suggested "refer to" over "see" (accessibility).

**Pull request merged**